### PR TITLE
feat: create a share link from an id

### PR DESCRIPTION
**Pull request by enguerran**
Need for https://github.com/cozy/cozy-photos-v3/pull/67

### CODE REVIEW

**Comment by enguerran** [comment]
I will update this PR to add `tests`.
For now, we accept only one id from album, features come with needs.

File: src/files.js [hunk 1]
```diff
@@ -245,6 +245,26 @@ export function getFilePath (cozy, file = {}, folder) {
   return `${folderPath}${file.name}`
 }
 
+export function getShareLink (cozy, id) {
+  if (!id) {
+    return Promise.reject(Error('An id should be provided to create a share link'))
```
Comment (gregorylegarec) [inline src/files.js]: Why avoiding the `new` keyword ? Is something we should generalize in all cozy-client-js ?

Comment (enguerran) [inline src/files.js]: Oh, no, it's a mistake.
I rewrite from (copy/paste from https://github.com/cozy/cozy-client-js/pull/160/files#diff-ecdbeb0b77447177e8165fbaf62b5a09R238)

```js
throw Error('An id should be provided to create a share link')
```

to this code.

Comment (enguerran) [inline src/files.js]: IDK what is the best way.

**Comment by enguerran** [comment]
Mmmh, I forget to add tests.

**Comment by enguerran** [comment]
[(2) [PHO-28 Partage d'un album par lien] Générer un lien de partage](https://trello.com/c/QFFEMBdU/280-2-pho-28-partage-d-un-album-par-lien-generer-un-lien-de-partage)

**Pull request merged**